script/bootstrap

```diff
@@ -57,7 +57,12 @@ install_packages_apt()
     sudo apt-get install --no-install-recommends -y libdbus-1-dev
 
     # mDNS
-    sudo apt-get install --no-install-recommends -y libavahi-client3 libavahi-common-dev libavahi-client-dev avahi-daemon
+    sudo apt-get install --no-install-recommends -y libavahi-client3 libavahi-common-dev libavahi-client-dev
+
+    if [[ ${OTBR_MDNS} == "avahi" ]]; then
```

Comment:
It seems the `OTBR_MDNS` environment variable isn't available at this point. You may need to define the variable in the script with a default value so that in the CI you're able to specify the mDNS provider.

Something like:
```
OTBR_MDNS="${OTBR_MDNS:-mDNSResponder}"
```